### PR TITLE
[DEV-3367] GitHub action for visual-dev storyshots

### PR DESCRIPTION
**Pull request by jayden-chan**
## Description of the change

Added a GitHub action to run the Visual Dev Storyshot automated snapshot tests. The tests will only run in comparison mode (will not save new snapshots)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: [DEV-3367](https://saasquatch.atlassian.net/browse/DEV-3367)
- Process.st launch checklist: N/A
### Development

- [x] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [x] All tests related to the changed code pass in development

### Paperwork

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has a Jira number
- [x] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

[DEV-3367]: https://saasquatch.atlassian.net/browse/DEV-3367?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/visual-dev-storyshot.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Visual Dev Storyshot Tests
```
Comment (johanventer) [inline .github/workflows/visual-dev-storyshot.yml:1]: Doesn't this need to actually install a newer chromium version?

Comment (jayden-chan) [reply .github/workflows/visual-dev-storyshot.yml:1]: Yes

https://coda.io/d/_d02Npt019Re/Storyshots-Setup-and-Usage_sutHZ#_lu__9

Comment (jayden-chan) [reply .github/workflows/visual-dev-storyshot.yml:1]: Oh wait I misread your comment. I thought you were questioning whether it's necessary to install a newer Chromium.

The Chromium that comes with Puppeteer is outdated, so we just use the one that's bundled with the GH Action `ubuntu-latest` runner. I put a link to it alongside the Puppeteer environment variables config.

**Comment by noahwc** [comment]
Closing since we've migrated visual-dev to UIRC and switched testing approaches.

**Pull request closed without merging**